docs/v8-migration.md

```diff
@@ -342,31 +342,33 @@ The following syntax plugins are no longer needed, you can safely remove them fr
 
 ![medium](https://img.shields.io/badge/risk%20of%20breakage%3F-medium-yellow.svg)
 
-- `includes` and `excludes` respect renamed package names ([#15576](https://github.com/babel/babel/pull/15576))
+- `include` and `exclude` respect renamed package names ([#15576](https://github.com/babel/babel/pull/15576))
 
-  **Migration**: If `includes` or `excludes` contain any plugins mentioned in the [Packages Renames section](./v8-migration.md#renamed-packages), change it to the new name. For example,
+  **Migration**: If `include` or `exclude` contain any plugins mentioned in the [Packages Renames section](./v8-migration.md#renamed-packages), change it to the new name. For example,
 
   ```diff title="babel.config.json"
   {
     "presets": [[
       "@babel/preset-env",
       {
-  -     "includes": ["proposal-optional-chaining"]
-  +     "includes": ["transform-optional-chaining"]
+  -     "include": ["proposal-optional-chaining"]
+  +     "include": ["transform-optional-chaining"]
       }
     ]]
   }
   ```
 
 ![low](https://img.shields.io/badge/risk%20of%20breakage%3F-low-yellowgreen.svg)
 
-- Enable the [`bugfixes`](./preset-env.md#bugfixes) option by default ([#13866](https://github.com/babel/babel/pull/13866))
+- The `bugfixes` options has been removed ([#13866](https://github.com/babel/babel/pull/13866), [#17078](https://github.com/babel/babel/pull/17078))
 
-  **Migration**: You will probably be fine with the new behaviour as Babel now tries to compile the broken syntax to the closest _non-broken modern syntax_ supported by your target browsers. If anyhow you want to restore the Babel 7 behaviour, you can specify `bugfixes: false`.
+  **Migration**: You will probably be fine with the new behaviour as Babel now tries to compile the broken syntax to the closest _non-broken modern syntax_ supported by your target browsers.
 
-- Removed syntax plugins can not be used in `includes` and `excludes` ([#15810](https://github.com/babel/babel/pull/15810))
+  If anyhow you want to restore the Babel 7 behaviour for debug purpose, you can specify the [`exclude` option](./preset-env.md#exclude): `{ exclude: [/bugfix/] }` to explicitly exclude the bugfixes plugins. Note that doing so will bloat the output as Babel has to compile features to older targets.
```

Comment:
Will disabling the bugfix cause the output to become larger?

Reply:
Yes, for example, given the input
```js
var a;
a?.b
```
with target `chrome: 90`,
the output with `{ bugfixes: true }` is
```js
var a;
a?.b;
```
whereas the output with `{ bugfixes: false }` is
```js
var a;
a === null || a === void 0 ? void 0 : a.b;
```